packages/gatsby-cli/src/init-starter.js

```diff
@@ -8,11 +9,15 @@ const report = require(`./reporter`)
 const url = require(`url`)
 const existsSync = require(`fs-exists-cached`).sync
 const { trackCli, trackError } = require(`gatsby-telemetry`)
+const prompts = require(`prompts`)
 const spawn = (cmd: string, options: any) => {
   const [file, ...args] = cmd.split(/\s+/)
   return execa(file, args, { stdio: `inherit`, ...options })
 }
 
+const pkg = require(`../package.json`)
+const conf = new Configstore(pkg.name, {})
```

Comment:
We already have configstore in place in our telemetry package and it's probably ok to reuse that file.
https://github.com/gatsbyjs/gatsby/blob/f5a85af786e2be73afcaf94f8da68f1e08bb42f9/packages/gatsby-telemetry/src/event-storage.js#L19

```suggestion
const conf = new Configstore(`gatsby`, {}, { globalConfigPath: true })
```

we probably want to abstract it away.